subversion.yaml

```
@@ -82,6 +90,16 @@ subpackages:
           mv "${{targets.destdir}}"/usr/lib/* "${{targets.subpkgdir}}"/usr/lib
     description: subversion libraries
 
+  - name: subversion-perl
+    description: perl bindings for subversion
+    dependencies:
+      runtime:
+        - subversion-libs
+    pipeline:
+      - uses: autoconf/make-install
+        with:
+          opts: pure_vendor_install -C subversion/bindings/swig/perl/native PERL_INSTALL_ROOT=${{targets.subpkgdir}}
```

```suggestion
          opts: pure_vendor_install -C subversion/bindings/swig/perl/native PERL_INSTALL_ROOT=.
```

Pretty sure this is relative to DESTDIR